hack/common.sh

```diff
@@ -95,7 +95,7 @@ METRICS_EXPORTER_FULL_IMAGE_NAME="${METRICS_EXPORTER_FULL_IMAGE_NAME:-${DEFAULT_
 
 DEFAULT_NOOBAA_BUNDLE_REGISTRY="quay.io/noobaa"
 DEFAULT_NOOBAA_BUNDLE_NAME="noobaa-operator-bundle"
-DEFAULT_NOOBAA_BUNDLE_TAG="v5.12.0"
+DEFAULT_NOOBAA_BUNDLE_TAG="master-20230306"
```

Comment:
Lets not change this one, This was created on request. It had the OB/OBC inside the bundle which is required. The default one does nit contain OB/OBC CRD's.

Reply:
Oh, Then I will revert this one